doc/ongoing_operations.rst

```diff
@@ -134,9 +134,32 @@ Disable all jobs related to a ROS distribution
 
 The process is the same as for for disabling a specific target.
 The prefixes are just slightly more generic to match all targets of that ROS distribution:
- * ``Lsrc_`` which matches all Lunar source jobs.
- * ``Lbin_`` which matches all Lunar binary jobs.
- * ``Lrel_`` which matches the Lunar release related management jobs.
- * ``Ldev_`` which matches the Lunar devel jobs as well as the management related jobs.
- * ``Ldoc_`` which matches the Lunar doc jobs as well as the management related jobs.
- * ``Lpr_`` which matches the Lunar PR jobs as well as the management related jobs.
+* ``Lsrc_`` which matches all Lunar source jobs.
+* ``Lbin_`` which matches all Lunar binary jobs.
+* ``Lrel_`` which matches the Lunar release related management jobs.
+* ``Ldev_`` which matches the Lunar devel jobs as well as the management related jobs.
+* ``Ldoc_`` which matches the Lunar doc jobs as well as the management related jobs.
+* ``Lpr_`` which matches the Lunar PR jobs as well as the management related jobs.
```

Comment:
Please avoid unrelated whitespace changes like this. All other files use a space to indent the bullets.

Reply:
Without this change the bullet list is displayed as commented out. That doesn't seem to be what we want here. I can submit it as a separate PR if you prefer